doc/Administration-guide/System-installation/Single-node-installation/index.md

```diff
@@ -65,18 +107,81 @@ Kaa requires the following third party components to be installed and configured
    In case of successful installation, you will receive the JDK version.
 
    ```bash
-   javac 1.8.0_66
+   javac 1.8.0_91
+   ```
+
+
+
+3. Install [MariaDB 5.5](https://mariadb.com/).
```

Comment:
I recommend to mention that user has option to install one of two SQL DBs. In addition this step can be presented as two tabs - one for MariaDB, another one for PostgreSQL.